http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -149,9 +149,11 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
         )
 
         val cmdToExecute =
-          s"echo \"${contentSecurityPolicyConfig.asString}\" > csp.txt && bash azure_vm_init_script.sh ${arguments
+          s"echo \"${contentSecurityPolicyConfig.asString}\" > csp.txt && bash azure_vm_init_script.sh ${
+            arguments
               .map(s => s"'$s'")
-              .mkString(" ")}"
+              .mkString(" ")
+          } > /var/log/azure_vm_init_script.log"
```

Comment:
Why do you need the env_vars.text? If you echo all variables they should already show up in this log file no?

Reply:
This was added by Justin - I guess this is a good point!

Do we have a preference? I think Justin's solution may make more sense.

Reply:
Although - this doesn't seem to label the arguments. 
Maybe I change my init script output to also be in /var/log, and name the file better? Such as `azure_vm_init_script_args.txt`

Reply:
That would be my preferred option as well, as long as this log file is indeed hidden from the user

Reply:
It appears as it stands `/var/log/` is available to the user to read.

What is the concern with them being able to see this info?

Reply:
I think I'd prefer to keep the logs, but I'll just restrict access to the log dump to read only for root. Since we have SSH capabilities now, this would make the debugging process a LOT easier. It's not easy to update this script, especially in prod.

Reply:
Sure, as long as the access is restricted and user cannot see it, feel free to keep it in

Reply:
@LizBaldo See screenshots - users can't see the log output. File is owned by root, and only root can read! So when we SSH, we have access still!

Reply:
Amazing, thanks so much! 👍